src/Microsoft.SqlTools.ServiceLayer/DacFx/Contracts/DacFxDeployRequest.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Microsoft.SqlTools.Hosting.Protocol.Contracts;
+using Microsoft.SqlTools.ServiceLayer.TaskServices;
+using Microsoft.SqlTools.ServiceLayer.Utility;
+
+namespace Microsoft.SqlTools.ServiceLayer.DacFx.Contracts
+{
+    /// <summary>
+    /// Parameters for a DacFx deploy request.
+    /// </summary>
+    public class DacFxDeployParams : IScriptableRequestParams
+    {
+        /// <summary>
+        /// Gets or sets connection string of the target database the deploy operation will run against.
+        /// </summary>
+        public string ConnectionString { get; set; }
+
+        /// <summary>
+        /// Gets or sets dacpac package filepath
+        /// </summary>
+        public string PackageFilePath { get; set; }
+
+        /// <summary>
+        /// Gets or sets name for deployed database
+        /// </summary>
+        public string TargetDatabaseName { get; set; }
+
+        /// <summary>
+        /// Connection uri
+        /// </summary>
+        public string OwnerUri { get; set; }
```

Comment:
seems redundant with the ConnectionString?

Reply:
Will remove ConnectionString